unit-tests/src/test/java/org/eclipse/collections/impl/list/immutable/ImmutableSingletonListTest.java

```diff
@@ -28,31 +29,31 @@ protected ImmutableList<Integer> classUnderTest()
     public void min_null_throws()
     {
         // Collections with one element should not throw to emulate the JDK Collections behavior
-        super.min_null_throws();
+        assertDoesNotThrow(() -> this.classUnderTestWithNull().min(Integer::compareTo));
```

Comment:
This one is interesting.. 
1) Junit4 doesn't have assertDoesNotThrow equivalent so w're using junit5 apis which isn't great but not a big ideal imo - i'm about to flip the imports soon
2) This test doesn't quite do what it says implemented in this way. The implementation is consistent with the comment but not with the test method name. Happy to just move to **// Not applicable**

Reply:
The only benefit that I see is that you explicitly declare your expectation - test isn't meant to throw an exception. Ideally every test should have some type of assertion so that one never wonders if this test was left incomplete for some reason.

Having said that, no strong opinion and I'm happy to remove the assertions if you like.